src/index.js

```diff
@@ -51,16 +51,37 @@ class App {
     this.container.appendChild(addTask);
     this.container.appendChild(this.taskListContainer);
     this.container.appendChild(buttonDeleteCompletedTasks);
+
+    if (localStorage.toDoTaskList.length) {
+      this.taskList = this.getTaskListFromLocalStorage();
+      this.render();
+    } else {
+      this.taskList = [];
+    }
   }
 
+  saveTaskListToLocalStorage() {
+    if (this.taskList.length) {
+      const taskListStr = JSON.stringify(this.taskList);
+      localStorage.setItem('toDoTaskList', taskListStr);
+    }
+  }
+
+  getTaskListFromLocalStorage() {
+    const dataFromLocalStorage = localStorage.getItem('toDoTaskList');
```

Comment:
a key can be moved to ex. this.appKey in constructor

Reply:
done